api/viewsets.py

```diff
@@ -92,6 +94,9 @@ class DocumentationLinkViewSet(
 class InfractionViewSet(ModelViewSet):
     serializer_class = InfractionSerializer
     queryset = Infraction.objects.all()
+    filter_backends = (DjangoFilterBackend, SearchFilter)
+    filter_fields = ('user__id', 'actor__id', 'active', 'hidden', 'type')
+    search_fields = ('$reason',)
```

Comment:
Do you think we should add a GIN index on this field? That could help us tremendously with infractions search. It might be better off to hold this off until later, though. See https://www.postgresql.org/docs/11/textsearch-tables.html#TEXTSEARCH-TABLES-INDEX.

Reply:
Sure, sounds like a good idea.